### PR TITLE
feat: add Kebab type to DropdownButton

### PR DESCRIPTION
**Pull request by natalieweesh**
### Overview

add a new kebab type button option to DropdownButton component

for use on WorkspaceCards (see corresponding PR here: https://github.com/codecademy-engineering/Codecademy/pull/24698) 🍡

### PR Checklist

- [ ] Related to designs: [figma](https://www.figma.com/file/hPv0Q1QJZa3nfcxsmc8dHV/Build?node-id=72%3A4133)
- [ ] Related to JIRA ticket: EGG-1261
- [ ] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

ezgif-6-6fcb4076d122

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.112-alpha.50922c.0<br/>@codecademy/gamut-labs@18.2.0-alpha.50922c.0<br/>@codecademy/styleguide@39.0.5-alpha.50922c.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60abdd7ce54b7500e56befe5--gamut-preview.netlify.app>https://60abdd7ce54b7500e56befe5--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60abdd7ce54b7500e56befe5>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.112-alpha.3b5d7e.0<br/>@codecademy/gamut-labs@18.2.0-alpha.3b5d7e.0<br/>@codecademy/styleguide@39.0.5-alpha.3b5d7e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60abe06cdd843f008c7962e5--gamut-preview.netlify.app>https://60abe06cdd843f008c7962e5--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60abe06cdd843f008c7962e5>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.113-alpha.85f65e.0<br/>@codecademy/gamut-labs@18.2.0-alpha.85f65e.0<br/>@codecademy/styleguide@40.0.1-alpha.85f65e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60ad1f0f71f8ed00cb08ce8f--gamut-preview.netlify.app>https://60ad1f0f71f8ed00cb08ce8f--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60ad1f0f71f8ed00cb08ce8f>Deploy Logs</a></p>

File: packages/gamut-labs/src/DropdownButton/index.tsx [hunk 1]
```diff
@@ -64,6 +67,18 @@ export const DropdownButton: React.FC<DropdownButtonProps> = ({
         </StrokeButton>
       );
       break;
+    case 'kebab':
+      clickTarget = (
+        <IconButton
+          icon={MiniKebabMenuIcon}
+          size="small"
+          mode="light"
```
Comment (codecaaron) [inline packages/gamut-labs/src/DropdownButton/index.tsx:75]:
```suggestion
```

This defaults to the current mode context so we don't need to set this! :D 

Also this adds some future proofing to make sure this works in light and dark mode by default :)

**Review by codecaaron — CHANGES_REQUESTED** [review]
Just the `mode="light"` removal and this looks good to go :)

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.114-alpha.b289dd.0<br/>@codecademy/gamut-labs@18.2.0-alpha.b289dd.0<br/>@codecademy/styleguide@40.0.2-alpha.b289dd.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60ad2b5b1ec7f100b7fe4523--gamut-preview.netlify.app>https://60ad2b5b1ec7f100b7fe4523--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60ad2b5b1ec7f100b7fe4523>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.114-alpha.df4c93.0<br/>@codecademy/gamut-labs@18.2.0-alpha.df4c93.0<br/>@codecademy/gamut@33.1.1-alpha.df4c93.0<br/>@codecademy/styleguide@40.0.2-alpha.df4c93.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.115-alpha.af66db.0<br/>@codecademy/gamut-labs@18.2.0-alpha.af66db.0<br/>@codecademy/styleguide@40.0.3-alpha.af66db.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60ad6d722801dc008ff5340e--gamut-preview.netlify.app>https://60ad6d722801dc008ff5340e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60ad6d722801dc008ff5340e>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.116-alpha.bef6ca.0<br/>@codecademy/gamut-labs@18.2.0-alpha.bef6ca.0<br/>@codecademy/styleguide@40.0.4-alpha.bef6ca.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60ad72a05dd9ea00883e6521--gamut-preview.netlify.app>https://60ad72a05dd9ea00883e6521--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60ad72a05dd9ea00883e6521>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.117-alpha.d9a281.0<br/>@codecademy/gamut-labs@18.2.0-alpha.d9a281.0<br/>@codecademy/styleguide@40.0.5-alpha.d9a281.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60ad90ae79bd96008fb3a265--gamut-preview.netlify.app>https://60ad90ae79bd96008fb3a265--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60ad90ae79bd96008fb3a265>Deploy Logs</a></p>

**Review by codecaaron — APPROVED** [review]

**Pull request merged**